bundles/org.openhab.binding.amazonechocontrol/src/main/resources/OH-INF/i18n/amazonechocontrol.properties

```diff
@@ -163,10 +163,18 @@ channel-type.amazonechocontrol.subtitle1.label = Subtitle 1
 channel-type.amazonechocontrol.subtitle1.description = Subtitle 1
 channel-type.amazonechocontrol.subtitle2.label = Subtitle 2
 channel-type.amazonechocontrol.subtitle2.description = Subtitle 2
+channel-type.amazonechocontrol.relativeHumidty.label = Humidity
+channel-type.amazonechocontrol.relativeHumidty.description = Humidity
```

Comment:
This file has to be regenerated after the description for humidity was updated.

Reply:
This has been corrected.